mne/preprocessing/stim.py

```diff
@@ -67,3 +91,90 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact(inst, events=None, event_id=None, tmin=0.,
+                      tmax=0.01, mode='linear', copy=False):
+    """Eliminates stimulations artifacts from instance
+
+    The instance will be modified in place (no copy)
+
+    Parameters
+    ----------
+    inst : instance of evoked or epochs
+        instance
+    events : array, shape (n_events, 3)
+        The list of events. No need when inst is epochs or evoked
+    event_id : int
+        The id of the events generating the stimulation artifacts.
+        No need when inst is epochs or evoked
+    tmin : float
+        Start time of the interpolation window in seconds.
+    tmax : float
+        End time of the interpolation window in seconds.
+    mode : 'linear' | 'window'
+        way to fill the artifacted time interval.
+        'linear' does linear interpolation
+        'window' applies a (1 - hanning) window.
+    copy : bool
+        If False inst is cropped in place.
+
+    Returns
+    -------
+    inst : instance of evoked or epochs
+        instance with modified data
+    """
+    if copy:
+        inst = inst.copy()
+    s_start = int(np.ceil(inst.info['sfreq'] * tmin))
+    s_end = int(np.ceil(inst.info['sfreq'] * tmax))
+    if (s_end - s_start - 4) < 0:
```

Comment:
@agramfort For this condition, I've not yet fixed. I think if it has lenght less than 4, it doesn't need to have window.. But I'm sure what I think.